Autoloads/Globals.gd

```diff
@@ -54,3 +70,27 @@ func open_settings_menu():
 		get_tree().root.add_child(settings_menu)
 	else:
 		push_warning('settings menu already exists in this scene')
+
+func set_window_mode(_value: int):
```

Comment:
Along with the other comment, I thinking these window/monitor utility functions probably belong in a separate script dedicated just to these things, to keep Globals light and acting as the manager of this stuff, in the same way it stores references to save data and user prefs, but leaves the internals to those specific scripts/classes.

Reply:
moved to Globals.display_manager.set_window_mode()